spec/models/user_spec.rb

```diff
@@ -66,6 +66,31 @@
       end
     end
 
+    describe '#check_biography_length' do
+      it 'biography can not have more than 150 words' do
+        # Text with 151 words
+        long_text = <<-EOS
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean
+          vestibulum, augue ut accumsan feugiat, mauris eros accumsan nunc,
+          volutpat vulputate eros orci quis nulla. Cum sociis natoque penatibus
+          et magnis dis parturient montes, nascetur ridiculus mus. Sed varius
+          orci ut lectus convallis, et ultrices ex finibus. Praesent orci augue,
+          aliquet at cursus at, placerat id ligula. Vestibulum a mauris non
+          felis pretium laoreet. Cras vel nisl convallis, pharetra ipsum at,
+          mattis erat. Praesent in lectus felis. Fusce eros mauris, euismod
+          lobortis metus id, tristique scelerisque nisl. Suspendisse potenti.
+          Suspendisse ac metus magna. Integer lobortis pharetra eros euismod
+          fringilla. Phasellus vitae orci vel magna laoreet mattis non eu neque.
+          Mauris ac dictum leo. Nullam dapibus convallis molestie. Integer
+          dignissim massa at odio feugiat tempus. Pellentesque ultrices rutrum
+          eros, a pellentesque lorem auctor in. Suspendisse sollicitudin dolor
+          vitae justo dignissim, a condimentum turpis molestie. Aenean
+          scelerisque, arcu eu congue mollis, nibh nulla finibus.
+        EOS
+        expect{create(:user, biography: long_text)}.to_not change { User.count }
```

Comment:
I think we can make it more apparant that it is a validation test if it was something like:
```Ruby
expect(build(:user, biography: long_text)).to_not be_valid
```
And please put it along with other [validation tests](https://github.com/Ana06/osem/blob/biography/spec/models/user_spec.rb#L14). It is a method indeed, but I think it is a validation method.